libs/core/langchain_core/runnables/graph.py

```diff
@@ -213,10 +229,38 @@ def trim_last_node(self) -> None:
                 self.remove_node(last_node)
 
     def draw_ascii(self) -> str:
-        return draw(
+        return draw_ascii(
             {node.id: node_data_str(node) for node in self.nodes.values()},
             [(edge.source, edge.target) for edge in self.edges],
         )
 
     def print_ascii(self) -> None:
         print(self.draw_ascii())  # noqa: T201
+
+    @overload
+    def draw_png(
+        self,
+        output_file_path: str,
+        fontname: Optional[str] = None,
+        labels: Optional[LabelsDict] = None,
+    ) -> None:
+        ...
+
+    @overload
+    def draw_png(
+        self,
+        output_file_path: None,
```

Comment:
FWIW i think we should be using `*` much systematically -- some of ours APIs end up with breaking changes b/c of missing `*` operators